### PR TITLE
An example OSGi bundle

### PR DESCRIPTION
**Pull request by OndroMih**

### CODE REVIEW

**Comment by mulderbaba** [comment]
I agree on @smillidge but I'm totally against the reverse, having modules know their parents.

File: OSGi/bundle-with-package-dependency/pom.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Activator>fish.payara.bundle.Activator</Bundle-Activator>
+                        <Import-Package>org.eclipse.persistence.expressions,
+                            org.osgi.framework;version="[1.5,2)"</Import-Package>
+                        <Bundle-SymbolicName>Payara-Example</Bundle-SymbolicName>
```
Comment (pdudits) [inline OSGi/bundle-with-package-dependency/pom.xml]: It is recommended to treat symbolic name like a package (or automatic module name in recent days), therefore `fish.payara.bundle` is more fitting name

**Comment by OndroMih** [comment]
> Should this be linked into the parent pom as a module?
I added the osgi module to parent pom. The osgi module doesn't refer to the parent pom and can still be built separately.

I also renamed the OSGi folder to osgi and added a parent osgi module.

File: OSGi/bundle-with-package-dependency/pom.xml (hunk 1 above)
Comment (OndroMih) [reply OSGi/bundle-with-package-dependency/pom.xml]: I changed both package name and symbolic name to `fish.payara.examplebundle`

**Pull request merged**